### ISSUE TITLE
PSR2 incorrect fix when multiple use statements on same line do not have whitespace between them

### ISSUE THREAD

**Issue by oklqh** [squizlabs/PHP_CodeSniffer#3053]
**Describe the bug**
phpcbf can not process 'use' keyword correctly when i use mulitiple namespaces seperated by ','.

**Code sample**
```php
<?php
namespace QL;

use phpQuery,Exception,ReflectionClass;
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcbf  --standard=psr2  --extensions=php test.php`
3. Content of  test.php became: 
```php
<?php
namespace QL;

use phpQuery;

useException,ReflectionClass;
```

**Expected behavior**
I hope it would be:
```php
<?php
namespace QL;

use phpQuery;
use Exception;
use ReflectionClass;
```

**Versions (please complete the following information):**
 - OS: CentOS Linux release 7.6.1810 (Core)
 - PHP: 7.1.7
 - PHPCS: PHP_CodeSniffer version 3.5.7 (stable)
 - Standard: PSR2

**Comment by oklqh** [comment]
it's easy reproducted. nobody want to fixe this bug?

**Comment by gsherwood** [comment]
Thanks for the bug report. I've committed a fix, which will be released in 3.5.7.
